docs/howtos/DataPlane Generation - DPG/07tcgcTypes.mdx

```diff
@@ -973,15 +989,18 @@ These are parameters to client initialization and method son the client. These w
 
 #### SdkEndpointParameter
 
-An `SdkEndpointParameter` represents a parameter to a client's endpoint. If the client is not parameterized, the endpoint parameter represents the entire endpoint input for the client. For parameterized clients, there will be as many `SdkEndpointParameter`s as needed to parameterize the client endpoint.
+An `SdkEndpointParameter` represents a parameter to a client's endpoint.
 
-```ts
+TCGC will always give it to you as overridable: If the server URL is a constant, we will return a templated endpoint with a default value of the constant server URL.
+In the case where the endpoint has extra template arguments, the type is a union of a completely-overridable endpoint, and an endpoint that accepts template arguments.
```

Comment:
We should also document the behavior when there are multiple servers - one with a default value and one with templated url.